Unity2DPlatformer/Assets/Scripts/Menu_Register.cs

```diff
@@ -66,61 +61,47 @@ bool validateEmail(string emailToValidate)
     public void createAccountButtonClicked()
     {
         dialogTextField.SetActive(false);
-        bool formValidateOk = true;
-
         getCredentialsFromForm();
 
         if (m_login == "")
         {
             m_dialogText = "Login cannot be empty!";
-            formValidateOk = false;
         }
         else if (m_password == "" || m_rePassword == "")
         {
             m_dialogText = "Passwords cannot be empty!";
-            formValidateOk = false;
         }
         else if (m_password.Length < 6)
         {
             m_dialogText = "Password has to have at least 6 characters!";
-            formValidateOk = false;
         }
         else if (m_password != m_rePassword)
         {
             m_dialogText = "Passwords are not equal!";
-            formValidateOk = false;
         }
         else if (!validateEmail(m_email))
         {
             m_dialogText = "Email is not correct!";
-            formValidateOk = false;
         }
         else
         {
-            // TODO Try creating account in DB
-
-            /* bool databaseError = false;
-             if (databaseError)
-             {
-                 errorString = "Something went wrong with database connection!";
-                 formValidateOk = false;
-             }*/
-
-            m_dialogText = "Account has been created, now you can log in!";
+            StartCoroutine(registerUser(m_login, m_password, m_email));
         }
+    }
 
-        if (formValidateOk)
-        {
-            // Remove credentials from form
-            m_login = "";
-            m_email = "";
-            m_password = "";
-            m_rePassword = "";
-        }
+    private IEnumerator registerUser(string login, string password, string email)
+    {
+        Database_Utils databaseUtils = new Database_Utils(DatabaseConnectionType.Php);
```

Comment:
Delete DatabaseConnectionType.Php and change default constructor in Database_Utils to PhpConnection